languages/ruby/main.go

```diff
@@ -3,7 +3,7 @@ package main
 // USING_CGO
 
 /*
-#cgo pkg-config: ruby-2.5
+#cgo CGO_LDFLAGS_ALLOW="-Wl,--compress-debug-sections=zlib" pkg-config: --libs --cflags ruby-2.7
```

Comment:
They added some security in the newer version of Go and this flags tells CGO that the flags given by pkg-config for the compiler are safe.